Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift

```diff
@@ -107,17 +107,16 @@ public class SentryReplayOptions: NSObject, SentryRedactOptions {
     public var unmaskedViewClasses = [AnyClass]()
 
     /**
-     * Enables the up to 5x faster experimental view renderer used by the Session Replay integration.
+     * Enables the up to 5x faster new view renderer used by the Session Replay integration.
      *
      * Enabling this flag will reduce the amount of time it takes to render each frame of the session replay on the main thread, therefore reducing 
      * interruptions and visual lag. [Our benchmarks](https://github.com/getsentry/sentry-cocoa/pull/4940) have shown a significant improvement of
-     * **up to 4-5x faster rendering** (reducing `~160ms` to `~36ms` per frame).
+     * **up to 4-5x faster rendering** (reducing `~160ms` to `~36ms` per frame) on older devices.
      *
-     * - Experiment: This is an experimental feature and is therefore disabled by default. In case you are noticing issues with the experimental
-     *               view renderer, please report the issue on [GitHub](https://github.com/getsentry/sentry-cocoa). Eventually, we will
-     *               remove this feature flag and use the experimental view renderer by default.
+     * - Experiment: In case you are noticing issues with the new view renderer, please report the issue on [GitHub](https://github.com/getsentry/sentry-cocoa). 
+     *               Eventually, we will remove this feature flag and use the new view renderer by default.
      */
-    public var enableExperimentalViewRenderer = false
+    public var enableViewRendererV2 = true
```

Comment:
Might be worth considering only deprecating `enableExperimentalViewRenderer`, making it an alias for `enableViewRendererV2`, so we don't break apps which used it.

Reply:
Added the `enableExperimentalViewRenderer` as an deprecated alias back for `enableViewRendererV2`